internal/clientsholder/clientsholder.go

```diff
@@ -58,7 +63,41 @@ func SetupFakeOlmClient(olmMockObjects []runtime.Object) {
 // For other (OLM, )
 // runtime mocking objects loading, use the proper clientset mocking function.
 func GetTestClientsHolder(k8sMockObjects []runtime.Object, filenames ...string) *ClientsHolder {
-	clientsHolder.K8sClient = k8sFakeClient.NewSimpleClientset(k8sMockObjects...)
+	// Build slices of different objects depending on what client
```

Comment:
Looks good! Would it be possible to also include the setup of the `clientsHolder.OlmClient`? This way we can also get rid of the `SetupFakeOlmClient()` function.

Reply:
I can do this in a separate PR to keep the changes separate.